examples/simulations/lorenz_ode/CMakeLists.txt

```diff
@@ -0,0 +1,27 @@
+#------------------------------------------------------------------------------#
+# Distributed under the OSI-approved Apache License, Version 2.0.  See
+# accompanying file Copyright.txt for details.
+#------------------------------------------------------------------------------#
+
+cmake_minimum_required(VERSION 3.12)
+project(ADIOS2SimulationsLorenzOdeExample)
+
+if(NOT TARGET adios2::cxx11)
+  set(_components C CXX)
+
+  find_package(ADIOS2 2.7.1 REQUIRED COMPONENTS ${_components})
+  if(NOT ADIOS2_FOUND)
```

Comment:
This is unneeded:
`find_package(ADIOS2 2.7.1 REQUIRED COMPONENTS ${_components})`

will exit with error if adios is not found

Reply:
We will exit with an error, but if we compile this test as an individual project find_package is needed to make use of the ADIOS2_DIR, otherwise ADIOS2_DIR is useless.